[rocBLAS] [hipBLAS] CHANGELOG update for 7.2 (#2668)
ROCm/rocm-libraries: projects/hipblas/CHANGELOG.md

Documentation for hipBLAS is available at
[https://rocm.docs.amd.com/projects/hipBLAS/en/latest/](https://rocm.docs.amd.com/projects/hipBLAS/en/latest/).

## hipBLAS 3.2.0 for ROCm 7.2

> ### Changed

Reviewer: ```suggestion
### Resolved issues
```

> * Client memory use counts for environment HIPBLAS_CLIENT_RAM_GB_LIMIT were corrected and avoids false clang static analysis warnings.

Reviewer: ```suggestion
* Corrected client memory use counts for the `HIPBLAS_CLIENT_RAM_GB_LIMIT` environment variable.
* Fix to avoid false Clang static analysis warnings.
```

Reviewer 2: The `and` was important as two separate issues, maybe just move the lattern to new item then,
* avoid false clang static analysis warnings
Resolved issues is fine by me too

Reviewer: Sure. That sounds good. I'll update my comments.

> * Client memory use counts for environment HIPBLAS_CLIENT_RAM_GB_LIMIT were corrected and avoids false clang static analysis warnings.

Reviewer: Would this be better under Resolved issues? (Applies to the entry below as well).

Author: Changed to `Resolved issues`, and moved `Resolved issues` after `Optimized`
